runtime/manta/src/lib.rs

```diff
@@ -479,6 +469,36 @@ impl manta_collator_selection::Config for Runtime {
     type CanAuthor = AuraAuthorFilter;
 }
 
+parameter_types! {
+    pub const ProposalBond: Permill = Permill::from_percent(1);
+    pub const ProposalBondMinimum: Balance = 3 * MANTA;
+    pub const ProposalBondMaximum: Balance = 50 * MANTA;
+    pub SpendPeriod: BlockNumber = 6 * DAYS;
```

Comment:
IIRC we'd want manta to be a slower calamari, not sure what the correct factor is, but i'd consider bumpind this to 14 days ( similar for voting )

Reply:
Makes sense.